spec/features/post_show_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'rails_helper'
+RSpec.describe 'Post Show', type: :feature do
+  before(:each) do
+    @user1 = User.create(
+      name: 'Amarachi',
+      bio: 'Product Manager',
+      photo: 'http://product.com',
+      posts_counter: 0
+    )
+    @user2 = User.create(
+      name: 'Makkie',
+      bio: 'Italian Chef',
+      photo: 'http://makkie.com',
+      posts_counter: 5
+    )
+    @post1 = Post.create(
+      user: @user1,
+      title: 'First post',
+      text: 'This is my first post',
+      comments_counter: 0,
+      likes_counter: 0
+    )
+    Comment.create(post_id: @post1.id, user_id: @user1.id, text: 'What an amazing time to be alive!')
+    Comment.create(post_id: @post1.id, user_id: @user1.id, text: 'Welcome micronauts!')
+    Comment.create(post_id: @post1.id, user_id: @user1.id, text: 'Congrats all, award well deserved.')
+    Comment.create(post_id: @post1.id, user_id: @user2.id, text: 'Peace all round.')
+    Comment.create(post_id: @post1.id, user_id: @user2.id, text: 'Hello world Favour')
+    Like.create(post_id: @post1.id, user_id: @user1.id)
+    Like.create(post_id: @post1.id, user_id: @user2.id)
+  end
+  describe 'A specific post with title, comments and counter' do
+    it 'displays the user for post' do
+      visit user_post_url(@user1, @post1)
+      expect(page).to have_content('Here is a specific post for a given user')
+    end
+    describe 'a post index page' do
```

Comment:
- [ ] **[OPTIONAL]** It is not a good practice to embed a `describe` block within another `describe` block. I would recommend you move the inner one out of this one. If you wish to provide more description within a describe block you can use the `context` block instead.

- [ ] **[OPTIONAL]** I think this second test should be **post show page**, as the index page is handled in the previous file